setup/steps/09_deploy_via_modelservice.py

```diff
@@ -0,0 +1,693 @@
+    
+    # Routing section
+    service_port = ev.get("vllm_common_inference_port", "8000")
+    release = ev.get("vllm_modelservice_release", "")
+    route_enabled = ev.get("vllm_modelservice_route", "false")
+    model_id = ev.get("deploy_current_model_id", "")
+    model_id_label = ev.get("deploy_current_model_id_label", "")
+    
+    # Image details
+    image_registry = ev.get("llmd_image_registry", "")
+    image_repo = ev.get("llmd_image_repo", "")
+    image_name = ev.get("llmd_image_name", "")
+    image_tag = ev.get("llmd_image_tag", "")
+    main_image = get_image(image_registry, image_repo, image_name, image_tag, 0)
+    
+    # Proxy details
+    proxy_image_registry = ev.get("llmd_routingsidecar_image_registry", "")
+    proxy_image_repo = ev.get("llmd_routingsidecar_image_repo", "")
+    proxy_image_name = ev.get("llmd_routingsidecar_image_name", "")
+    proxy_image_tag = ev.get("llmd_routingsidecar_image_tag", "")
+    proxy_image = get_image(proxy_image_registry, proxy_image_repo, proxy_image_name, proxy_image_tag, 0)
+    proxy_connector = ev.get("llmd_routingsidecar_connector", "")
+    proxy_debug_level = ev.get("llmd_routingsidecar_debug_level", "")
+    
+    # EPP and routing configuration
+    inference_model_create = ev.get("vllm_modelservice_inference_model", "true")
+    inference_pool_create = ev.get("vllm_modelservice_inference_pool", "true")
+    epp_create = ev.get("vllm_modelservice_epp", "true")
+    
+    # Decode configuration
+    decode_replicas = int(ev.get("vllm_modelservice_decode_replicas", "0"))
+    decode_create = "true" if decode_replicas > 0 else "false"
+    decode_data_parallelism = ev.get("vllm_modelservice_decode_data_parallelism", "1")
+    decode_tensor_parallelism = ev.get("vllm_modelservice_decode_tensor_parallelism", "1")
+    decode_model_command = ev.get("vllm_modelservice_decode_model_command", "")
+    decode_extra_args = ev.get("vllm_modelservice_decode_extra_args", "")
+    decode_cpu_mem = ev.get("vllm_modelservice_decode_cpu_mem", "")
+    decode_cpu_nr = ev.get("vllm_modelservice_decode_cpu_nr", "")
+    decode_inference_port = ev.get("vllm_modelservice_decode_inference_port", "8000")
+    
+    # Prefill configuration
+    prefill_replicas = int(ev.get("vllm_modelservice_prefill_replicas", "0"))
+    prefill_create = "true" if prefill_replicas > 0 else "false"
+    prefill_data_parallelism = ev.get("vllm_modelservice_prefill_data_parallelism", "1")
+    prefill_tensor_parallelism = ev.get("vllm_modelservice_prefill_tensor_parallelism", "1")
+    prefill_model_command = ev.get("vllm_modelservice_prefill_model_command", "")
+    prefill_extra_args = ev.get("vllm_modelservice_prefill_extra_args", "")
+    prefill_cpu_mem = ev.get("vllm_modelservice_prefill_cpu_mem", "")
+    prefill_cpu_nr = ev.get("vllm_modelservice_prefill_cpu_nr", "")
+    
+    # Resource configuration
+    accelerator_resource = ev.get("vllm_common_accelerator_resource", "")
```

Comment:
The env var gets reset in `check_affinity()`; the value in `ev` does not.
```suggestion
    accelerator_resource = os.getenv("LLMDBENCH_VLLM_COMMON_ACCELERATOR_RESOURCE")
```